package.json

```diff
@@ -1,17 +1,46 @@
 {
-  "name": "tarball-extract",
+  "name": "tarball_extract",
   "description": "A simple tarball download and extraction lib for node.",
-  "version": "0.0.2",
-  "author": "Joshua Heiks <joshuaheiks@gmail.com>",
-  "keywords": ["tar", "tarball"],
+  "version": "0.0.3",
```

Comment:
Let's make the version 0.0.2-npnf.0.1  That will distinguish it from the original author's 0.0.3, if he releases such.